website/docs/docs/contributing/testing-a-new-adapter.md

```diff
@@ -74,7 +74,7 @@ models:
 
 </File>
 
-2. Use the "fixtures" to define the project for your test case. Following the default pytest configurations, the file name must begin with `test_`, and the class name must begin with `Test`.
+2. Use the "fixtures" to define the project for your test case. These fixtures are always scoped to the **class**, where the class represents one test case—that is, one dbt project or scenario. Following the default pytest configurations, the file name must begin with `test_`, and the class name must begin with `Test`.
```

Comment:
only thing I can think to add would be something about why for the basic tests there is only one method per test case?

Reply:
good call!